src/routes.tsx

```diff
@@ -3,15 +3,21 @@ import { hashHistory, Router, RouteConfig } from 'react-router';
 
 import { App } from './App';
 import { Home } from './home';
-import { NotFound } from './not-found/NotFound';
+import { NotFound } from './not-found';
+import StatelessComponent = React.StatelessComponent;
+import Component = React.Component;
```

Comment:
This namespace aliasing is wrong. Pls watch what we storm generates for your next time before committing